eng/pipelines/templates/steps/java-publishing.yml

```diff
@@ -16,7 +16,7 @@ steps:
         workingDirectory: $(Agent.BuildDirectory)
         filePath: ${{ parameters.JavaRepoRoot }}/eng/scripts/Publish-MavenPackages.ps1
         arguments: >
-          -ArtifactDirectory ${{ parameters.ArtifactDirector }}
```

Comment:
Wow how did this ever even work before?

Reply:
It's been there for a while.  I guess no one uses burner deployments

Reply:
Oh yeah I think this is dead code now as we stopped using burner feeds awhile ago. It might be worth removing after a quick search to confirm we aren't using it any longer.

Reply:
I'll remove this in a following refactor PR